src/main/java/no/seime/openhab/binding/esphome/internal/message/AbstractMessageHandler.java

```diff
@@ -90,6 +68,73 @@ protected ChannelType addChannelType(final String channelTypePrefix, final Strin
         return channelType;
     }
 
+    protected StateDescription readOnlyStateDescription() {
+        return StateDescriptionFragmentBuilder.create().withReadOnly(true).build().toStateDescription();
+    }
+
+    protected StateDescription addStateDescription(String pattern) {
```

Comment:
This method and others are named addXXX, but they don't add something, they create something. Suggest renaming to createXXX or buildXXX